modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -3009,6 +3012,8 @@ void GDScriptAnalyzer::reduce_identifier(GDScriptParser::IdentifierNode *p_ident
 			p_identifier->reduced_value = p_identifier->constant_source->initializer->reduced_value;
 			found_source = true;
 			break;
+		case GDScriptParser::IdentifierNode::MEMBER_SIGNAL:
+			[[fallthrough]];
 		case GDScriptParser::IdentifierNode::INHERITED_VARIABLE:
```

Comment:
Fallthrough doesn't need to be explicit if there's no specific code to run before the fallthrough.
```suggestion
		case GDScriptParser::IdentifierNode::MEMBER_SIGNAL:
		case GDScriptParser::IdentifierNode::INHERITED_VARIABLE:
```